site/pages/admin/add-user.page.tsx

```diff
@@ -0,0 +1,131 @@
+import TextInput from "../../components/form/TextInput";
+import { TwoThirdsLayout } from "../../components/layout/Layout";
+import { COOKIES_ADD_USER_ERRORS } from "../../constants";
+import { PageState } from "../../interfaces";
+import { AddUserSchema, addUserSchema } from "../../schemas/add-user.schema";
+import { destroyCookieOnResponseObject, getPageState } from "../../utils/apiUtils";
+import { getSessionWithOrgDetail } from "../../utils/apiUtils/auth";
+import { getStateUpdater } from "../../utils/formUtils";
+
+const title = "Add User - Create Transport Disruptions Service";
+const description = "Add User page for the Create Transport Disruptions Service";
+
+export interface AddUserPageProps extends PageState<Partial<AddUserSchema>> {}
+
+const AddUser = (props: AddUserPageProps): ReactElement => {
+    const [pageState, setPageState] = useState(props);
+
+    const stateUpdater = getStateUpdater(setPageState, pageState);
+
+    return (
+        <TwoThirdsLayout title={title} description={description} errors={pageState.errors}>
+            <CsrfForm action="/api/add-user" method="post" csrfToken={pageState.csrfToken}>
+                <>
+                    <ErrorSummary errors={pageState.errors} />
+                    <h1 className="govuk-heading-xl">Add new user</h1>
+                    <TextInput<AddUserSchema>
+                        display="First name"
+                        inputName="givenName"
+                        widthClass="w"
+                        value={pageState.inputs.givenName}
+                        initialErrors={pageState.errors}
+                        schema={addUserSchema.shape.givenName}
+                        stateUpdater={stateUpdater}
+                        maxLength={100}
+                    />
+                    <TextInput<AddUserSchema>
+                        display="Last name"
+                        inputName="familyName"
+                        widthClass="w"
+                        value={pageState.inputs.familyName}
+                        initialErrors={pageState.errors}
+                        schema={addUserSchema.shape.familyName}
+                        stateUpdater={stateUpdater}
+                        maxLength={100}
+                    />
+                    <TextInput<AddUserSchema>
+                        display="Email address"
+                        inputName="email"
+                        widthClass="w"
+                        value={pageState.inputs.email}
+                        initialErrors={pageState.errors}
+                        schema={addUserSchema.shape.email}
+                        stateUpdater={stateUpdater}
+                        maxLength={100}
+                    />
+
+                    <Table rows={[{ header: "Organisation", cells: [pageState.sessionWithOrg?.orgName, ""] }]} />
+                    <input type="hidden" name="orgId" value={pageState.sessionWithOrg?.orgId} />
```

Comment:
This should be retrieved from the session directly in the API call, otherwise people could pass through whatever orgId they want

Reply:
done. More details in comment above